components/admin_console/system_user_detail/team_list/index.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {getTeamsForUser, getTeamMembersForUser} from 'mattermost-redux/actions/teams';
+
+import {getCurrentLocale} from 'selectors/i18n';
+import {t} from 'utils/i18n';
+
+import TeamList from './team_list.jsx';
+
+function mapStateToProps(state) {
+    return {
+        emptyListTextId: t('admin.team_settings.team_list.no_teams_found'),
+        emptyListTextDefaultMessage: 'No teams found',
```

Comment:
I wonder if it's better to remove `emptyListTextId` and `emptyListTextDefaultMessage` as props from the container components (index.js) and move them into team_list.jsx since they're not using the Redux state directly? I don't think there's anything wrong with it I'm just wondering from a consistency perspective?

Reply:
That's probably a good idea for consistency.